scripts/tutorial-uploader/setup-for-testing.ts

```diff
@@ -0,0 +1,56 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2024.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { API } from "./lib/api";
+import { createItem } from "@directus/sdk";
+
+if (/learning-api\.quantum\.ibm\.com/.test(process.env.LEARNING_API_URL!)) {
+  throw new Error(
+    "Tried to run setup against production! Set the env var LEARNING_API_URL to either staging or local (see tutorial-uploader/README.md)",
+  );
+}
+
+const TOPICS = [
+  "Chemistry",
+  "Dynamic circuits",
+  "Error mitigation",
+  "Optimization",
+  "Qiskit Patterns",
+  "Scheduling",
+  "Transpilation",
+];
+
+const CATEGORIES = ["Workflow example", "How-to"];
```

Comment:
What's the impact of these becoming stale? If we use a new value in the config file that is not yet defined here, we'd fail in the test, which would be a signal to update this file?

Reply:
Yes, adding a category that doesn't exist would break the uploader. I'll make a check that verifies the list against production.